.gitignore

```diff
@@ -1,6 +1,183 @@
+
```

Comment:
Can we please not have this big `.gitignore` in every one of our repos ? This should probably be set up in a global .gitignore on your machine.
cf https://github.com/DataDog/datadogpy/pull/433#discussion_r343054954

Reply:
Moved to #88